airflow/docker_image.go

```diff
@@ -276,6 +276,91 @@ func (d *DockerImage) TagLocalImage(localImage string) error {
 	return nil
 }
 
+func (d *DockerImage) Run(dagID, envFile, settingsFile, containerName string, taskLogs bool) error {
+	// delete container
```

Comment:
```suggestion
```
nit: is it still relevant, or else we can drop it?

Reply:
yes we need to delete the container here just in case on exists and wasn't deleted before for some reason

Reply:
yeah, but the comment over here does not fit any purpose to have it at the start of `Run` function, makes sense to have it where we are deleting the container but that is already there